example/s3select_example.cpp

```diff
@@ -336,6 +336,11 @@ int run_query_on_parquet_file(const char* input_query, const char* input_file)
 
     std::cout << result << std::endl;
 
+    if(status == 2) // limit reached
```

Comment:
we should move from using numbers as different status states, and replace that with enum-class